src/stylesheets/components/_accordions.scss

```diff
@@ -19,24 +15,24 @@ $v-padding: $spacing-md-small;
 // scss-lint:disable PropertyCount
 @mixin accordion-button-styles {
   @include button-unstyled;
-  background-color: $color-gray-lightest;
+  background-color: color('gray-5');
   background-image: url('#{$theme-image-path}/minus.png');
   background-image: url('#{$theme-image-path}/minus.svg');
-  background-position: right $h-padding center;
+  background-position: right units(2.5) center;
   background-repeat: no-repeat;
-  background-size: $icon-size;
-  color: $color-base;
+  background-size: units(2);
+  color: $color-base; // theme-base or something?
   cursor: pointer;
   display: inline-block;
-  font-family: $font-sans;
-  font-weight: $font-bold;
+  font-family: $theme-font-sans;
```

Comment:
This won't work properly in this format. `$theme-font-sans` is a key/token/string (like `'source-sans-pro'`) rather than a value/stack (like `"Source Sans Pro", Helvetica, sans` or whatever).

I realize this is confusing (see https://github.com/uswds/uswds/issues/2633).

For now, use `@include u-font-family('sans')` — but I can make a function as well... something like `family()` as well as a var (`$font-family-sans`)